src/rasterstats/main.py

```diff
@@ -45,7 +45,8 @@ def gen_zonal_stats(
         zone_func=None,
         raster_out=False,
         prefix=None,
-        geojson_out=False, **kwargs):
+        geojson_out=False,
+        properties=None, **kwargs):
```

Comment:
Why an additional `properties` argument? The properties of the vector data should be available directly from the features.